Libraries/Tasks/Locking/ObjectLockHolder.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Concurrent;
+using Orchard;
+using Orchard.Environment.Extensions;
+
+namespace Piedone.HelpfulLibraries.Libraries.Tasks.Locking
+{
+    public interface IObjectLockHolder : ISingletonDependency
+    {
+        bool TryAcquire(string name);
+        void Release(string name);
+    }
+
+
+    [OrchardFeature("Piedone.HelpfulLibraries.Tasks.Locking")]
+    public class ObjectLockHolder : IObjectLockHolder
+    {
+        private readonly object _dictionaryLock = new object();
+        private ConcurrentDictionary<string, bool> _locks = new ConcurrentDictionary<string, bool>();

```

Comment:
This can be `readonly` too, no?

I also get this in VS 2022 
![image](https://github.com/Lombiq/Helpful-Libraries/assets/80963259/c7554501-df41-49e7-a943-f332ac83a0ce)

(OFC same in the other PR)

Reply:
Yep.